index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <p> test gps in body </p>
+    <a href="https://developers.google.com/maps/documentation/javascript/overview#javascript"> link to new api</a>
+    <div id="divId"></div>
+    <div id="map">
+
```

Comment:
Please pay attention to formatting and indentation, tags should be at the same level and preferably without unnecessary empty space https://www.w3schools.com/html/html5_syntax.asp